src/studio/src/designer/frontend/app-development/features/dataModeling/containers/DataModelingContainer.tsx

```diff
@@ -46,17 +58,96 @@ function DataModelingContainer(): JSX.Element {
   const onSaveSchema = (schema: any) => {
     dispatch(saveDataModel({ schema }));
   };
+  const onCreateClick = (event: any) => {
+    setCreateButtonAnchor(event.currentTarget);
+  }
+  const onNewModelNameChanged = (e: any) => {
+    setNewModelName(e.target.value);
+  }
+  const onCreateConfirmClick = (e: any) => {
+    if (newModelName && newModelName.length > 0) {
+      dispatch(newDataModel({ modelName: newModelName }));
+    }
+    setCreateButtonAnchor(null);
+    setNewModelName(null);
+  }
+  const onDeleteClick = (event: any) => {
+    setDeleteButtonAnchor(event.currentTarget);
+  }
+  const onDeleteConfirmClick = () => {
+    dispatch(deleteDataModel({}));
+    setDeleteButtonAnchor(null);
+  }
+  const onCancelDelete = () => {
+    setDeleteButtonAnchor(null);
+  }
+  const onCancelCreate = () => {
+    setCreateButtonAnchor(null);
+    setNewModelName(null);
+  }
 
   return (
     <div className={classes.root}>
-      <SchemaSelect id="schema" value={selectedDataModelName} onChange={onSchemaSelected}/>
-      <SchemaEditorApp
-        schema={jsonSchema || {}}
-        onSaveSchema={onSaveSchema}
-        rootItemId={`#/definitions/${selectedDataModelName}`}
-      />
+      <Grid container>
+        <Grid item>
+          <Button
+            variant="contained"
+            className={classes.button}
+            startIcon={<AddCircleOutline />}
+            onClick={onCreateClick}
+          >
+            New
+        </Button>
+        </Grid>
+        <Grid item xs={9}>
+          <SchemaSelect id="schema" value={selectedDataModelName} onChange={onSchemaSelected} />
+        </Grid>
+        <Grid item>
+          <Button
+            variant="contained"
+            className={classes.button}
+            startIcon={<DeleteOutline />}
+            onClick={onDeleteClick}
+          >
+            Delete
+          </Button>
+        </Grid>
+        <AltinnPopover
+          anchorEl={deleteButtonAnchor}
+          anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+          btnCancelText="Nei"
+          descriptionText={`Er du sikker på at du vil slette ${selectedDataModelName}?`}
```

Comment:
I miss internalization for texts. A language prop is passed to each subapp by default. Then you can use `getLanguageFromKey` from `app-shared/utils/language` to fetch languages keys defined in `nb.ini` and `en.ini`

Reply:
In this case (with a parameter as well) you could use the `getParsedLanguageFromKey` from the same file as Steffen mentions, setting up the text in the language file with a placeholder. Ex:
`Er du sikker på at du vil slette {0}`. Params are supplied to the function, and you can specify that you want a string output.